_sass/_aiml.scss

```diff
@@ -34,11 +33,16 @@
 }
 
 .ai-content-container{
-    display: grid;
-    grid-template-columns: 30% 70%;
+    //replace grid with flex box
+    // display: grid;
+    // grid-template-columns: 30% 70%;
+    display: inline-flex;
```

Comment:
Before we merge, make sure to remove the comments!

Reply:
ok i removed them